README.md

```diff
@@ -20,38 +30,13 @@ default:
   extensions:
     BehatExtension\DoctrineDataFixturesExtension\Extension:
       lifetime:    'feature'
-      directories: ~
-      fixtures:    ~
 ```
 
 When **lifetime** is set to "feature" (or unspecified), data fixtures are reloaded between feature files.  Alternately,
 when **lifetime** is set to "scenario", data fixtures are reloaded between scenarios (i.e., increased
 test isolation at the expense of increased run time).
 
-When **fixtures** is set, the extension will load the specified fixture classes.
-This must contain a list of fully qualified class names. Classes MUST not have any constructor arguments (or at least optional).
-If the interface `Symfony\Component\DependencyInjection\ContainerAwareInterface` is implemented, the container is set to the fixture loader.
-
-When **directories** is set, the extension will load the data fixtures globed from the respective directories.
-Classes MUST not have any constructor arguments (or at least optional).
-If the interface `Symfony\Component\DependencyInjection\ContainerAwareInterface` is implemented, the container is set to the fixture loader.
-
-This extension will also load every fixtures declared as services and tagged with `doctrine.fixture.orm`.
-
-```yaml
-# behat.yml
-default:
-  # ...
-  extensions:
-    BehatExtension\DoctrineDataFixturesExtension\Extension:
-      lifetime: 'feature'
-      directories:
-        - '/project/src/AcmeAnalytics/Tests/DataFixtures/ORM'
-      fixtures:
-        - 'Acme\StoreBundle\DataFixture\ORM\Categories'
-        - 'Acme\StoreBundle\DataFixture\ORM\Apps'
-        - 'Acme\VendorBundle\DataFixture\ORM\Vendors'
-```
+This extension will load every fixtures declared as services and tagged with `doctrine.fixture.orm`.
```

Comment:
The problem here is that the exension won't work anymore for non-Symfony applications